### ISSUE TITLE
Translated field's value change doesn't get reflected in Translated Term's Internal Source for Odoo's default language other than English (US)

### ISSUE THREAD

**Issue by blackwaremayank** [odoo/odoo#48543]
Impacted versions: 13

Steps to reproduce: 
**Issue #1** Create new Odoo Enterprise Database with Language English (CA)

Now install Sales Management module and also enable French (CA) language.

Now create one Product & add **Sales Description** as shown in below snapshot and Save it.
![image](https://user-images.githubusercontent.com/48239869/77784018-d7173000-707f-11ea-9e21-e0ea383673f4.png)

Now edit this same product & remove **Sales Description** and Save it.
![image](https://user-images.githubusercontent.com/48239869/77784168-1b0a3500-7080-11ea-8e2b-17c4f6854dd8.png)

But **Sales Description** will remain unchanged.

**Issue #2** I'm facing the same above issue with **Purchase Description** field as well.

![image](https://user-images.githubusercontent.com/48239869/77784734-05493f80-7081-11ea-970c-f62b9b866567.png)

**Issue #3** Facing the same issue for Translatable field in **Helpdesk Stages**.

![image](https://user-images.githubusercontent.com/48239869/77785453-39713000-7082-11ea-8366-c3b1f366263c.png)

**So basically every Translatable fields in each views have this issue.** 

Current behavior:

Record's Translatable field isn't updated properly as **Internal Source** of Translated Terms.

Expected behavior:

Record's Translatable field should be updated properly as Internal Source of Translated Terms.

![image](https://user-images.githubusercontent.com/48239869/77784955-68d36d00-7081-11ea-9212-c68fb744da40.png)

Video/Screenshot link (optional):

**Comment by pedrobaeza** [comment]
This is due to a limitation in the translation system: an empty translation means that the source term is got. Put an space for example for avoiding this.

Closing as known limitation. In v14 there might be some work on this, isn't it, @mart-e ?

**Comment by mart-e** [comment]
No actually this should be fixed in 13.0, there is even [a test about this](https://github.com/odoo/odoo/blob/b778d5a272e13a6b99eb0a9a7c324a0823946262/odoo/addons/base/tests/test_translate.py#L506). I could reproduce the problem on `product.template` but not on `res.country`. Maybe due to the `_inherits` ? To investigate...

**Comment by pedrobaeza** [comment]
OK, then reopening and assigning you.

**Comment by dirceu3h** [comment]
Just to let you know i am having the same issue guys, for now the blank space workaround is working for me. thanks @pedrobaeza and @Yenthe666
